### PR TITLE
Adds base html file that describes template of repo

### PR DESCRIPTION
**Pull request by KeenWarrior**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KeenWarrior) [inline index.html:3]: This is crap

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
```
Comment (KeenWarrior) [inline index.html:9]: Done

File: index.html (hunk 1 above)
Comment (RASHMIP2691) [reply index.html:3]: Please add the bold for the text

Comment (RASHMIP2691) [reply index.html:3]: Please add the bold for the text

**Pull request merged**